src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -35,6 +37,16 @@ object ProjectBuilder {
         (File(s"${name.toLowerCase}", "Dockerfile"), dockerfileContents)
     }
 
-    Project(databaseCreationScripts ++ dockerfiles)
+    val kubeFiles = KubernetesGenerator.generate(
+      KubernetesBuilder
+        .createServiceKubeFiles(
+          templefile.projectName,
+          (templefile.services.zip(1024 until Int.MaxValue) map {
```

Comment:
Adding a dot before map means you don’t need the surrounding parens, may make it nicer